tests/clusterclient_async_sequence.c

```diff
@@ -5,13 +5,15 @@
  * The behaviour is similar to that of clusterclient_async.c, but it sends the
  * next command after receiving a reply from the previous command.
  * It is still possible to send multiple commands at once like in
- # clusterclient_async.c using following action commands:
- #
- # !async - Send multiple commands and then wait for their responses.
- #          Will send all following commands until EOF or the command `!sync`
- #
- # !sync  - Send a single command and wait for its response before sending next
- #          command. This is the default behaviour.
+ * clusterclient_async.c using following action commands:
+ *
+ * !async  - Send multiple commands and then wait for their responses.
+ *           Will send all following commands until EOF or the command `!sync`
+ *
+ * !sync   - Send a single command and wait for its response before sending next
+ *           command. This is the default behaviour.
+ *
+ * !resend - Resend a failed command from its reply callback.
```

Comment:
Does `!resend` activate resending for all failed commands for the rest of the session?

Reply:
Updated text